scripts/jenkins/cloud/ansible/roles/deploy_caasp_terraform/tasks/deploy_caasp.yml

```diff
@@ -0,0 +1,348 @@
+#
+# (c) Copyright 2020 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- name: Management Machine | Update resolv.conf
+  shell:
+    echo "nameserver {{ dns_server_ip1 }}" | sudo tee -a /etc/resolv.conf
```

Comment:
This change gets lost when the lease is renewed. I did a PR for ardana so neutron's dhcp server pushes it out instead.

Reply:
Updated